### DISCUSSION TITLE
Pixel 6 - AR capabilities

### DISCUSSION THREAD

**Discussion by Jasonp8000** [google/model-viewer#2980] [Q&A]
Hi everyone,

I'm looking at purchasing a pixel 6, I have realised it has no ToF 3D depth sensor. Will the phone struggle to place objects on difficult surfaces such as walls and ceilings? 

Thanks in advance, 

Jason

**Comment by ghost** [comment]
Obviously a dedicated sensor is going to fare better overall, but I don't believe any Pixel phone has a dedicated depth sensor, personally I notice little to no practical difference between a Pixel phone and LiDAR equipped phone when it comes to plane detection in AR. I mostly use Pixel phones for testing though so I have probably unconsciously trained myself in the best techniques for detecting planes.
